lib/active_interaction/filters/file_filter.rb

```diff
@@ -4,8 +4,9 @@ module ActiveInteraction
   class Base
     # @!method self.file(*attributes, options = {})
     #   Creates accessors for the attributes and ensures that values passed to
-    #   the attributes respond to the `eof?` method. This is useful when passing
-    #   in Rails params that include a file upload or another generic IO object.
+    #   the attributes respond to the `rewind` method. This is useful when
+    #   passing in Rails params that include a file upload or another generic
+    #   IO object.
```

Comment:
What about listing examples like "File, Tempfile, and ActionDispatch::Http::UploadedFile"?